docs/proposals/formatted-output.md

```diff
@@ -10,12 +10,19 @@
   - [Scenarios](#scenarios)
     - [Scripting](#scripting)
     - [CI/CD](#cicd)
+    - [Verify local files](#verify-local-files)
   - [Proposal and desired user experience](#proposal-and-desired-user-experience)
     - [oras manifest fetch](#oras-manifest-fetch)
     - [oras pull](#oras-pull)
     - [oras push](#oras-push)
     - [oras attach](#oras-attach)
     - [oras discover](#oras-discover)
```

Comment:
None of the other commands have links to the format specific sections. Should this be scipped?

Reply:
I think the ToC is supposed to flatten all levels of titles? The other commands don't have format specific sections.